app/decorators.py

```diff
@@ -95,4 +95,4 @@ def validate_list_data(*args, **kwargs):
         elif items_data["item_name"] == "" or items_data["completed"] == "":
             return{"message": "You have to provide all the item name and the completed status"}, 400
         return func(*args, **kwargs)
-    return validate_list_data 
\ No newline at end of file
+    return validate_list_data
```

Comment:
Github will give you that red thing you can see there...if you hoover over it it will tell you the problem..